sdk/eventhub/perf-tests/event-hubs-track-1/README.md

```diff
@@ -0,0 +1,12 @@
+### Guide
+
+1. Navigate to `sdk\eventhub\perf-tests\event-hubs-track-1`
```

Comment:
I wonder if we have guidelines regarding what style to use for path connectors, like `/` or `\` 🤔

Reply:
Also, this line is missing a dot 🙈

Reply:
I'll do /

Reply:
```suggestion
1. Navigate to `sdk/eventhub/perf-tests/event-hubs-track-1`
```